_posts/en/newsletters/2020-08-26-newsletter.md

```diff
@@ -40,7 +40,23 @@ answers posted since our last update.*
 {% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
 {% assign bse = "https://bitcoin.stackexchange.com/a/" %}
 
-FIXME:bitschmidty
+- [What is the motivation for separating Bitcoin Core into independent node, wallet, and GUI processes?]({{bse}}98398)
+  Michael Folkson outlines the benefits of Russell Yanofsky's [efforts][bitcoin
+  core process separation] to separate Bitcoin Core into different `bitcoin-node`,
+  `bitcoin-wallet`, and `bitcoin-gui` processes. Modularity would allow users more
+  flexibility in their setups and developers maintainability and security benefits.
+
+- [What's the most efficient way to create a raw transaction with a specific fee rate?]({{bse}}98392)
+  User Darius asks about the interplay between UTXO selection and feerate when
+  constructing a transaction. Murch provides an overview of the effective value
+  approach to coin selection, consideration of [change avoidance][bitcoin wiki change avoidance],
+  and attention to signature length.
+
+- [Will there be a testnet4? Or do we not need a testnet reset once we have Signet?]({{bse}}98579)
+  Michael Folkson explains the considerations involving a potential testnet
+  reset or introduction of a new [signet][topic signet] network. A [related Stack
+  Exchange question][stack exchange signet setup] this month also summarizes two
+  options for working with signet.
```

Comment:
Probably out of scope, but a signet meta-issue was opened today by Michael as well and discussion has begun: https://github.com/bitcoin/bitcoin/issues/19787